contracts/Pair.sol

```diff
@@ -122,13 +122,13 @@ contract Pair is IPair {
         externalBribe = _externalBribe;
         _safeApprove(token0, externalBribe, type(uint).max);
         _safeApprove(token1, externalBribe, type(uint).max);
-        emit ExternalBribeSet(msg.sender, _externalBribe);
+        emit ExternalBribeSet(_externalBribe);
     }
 
     function setHasGauge(bool value) external {
         require(msg.sender == voter, 'Only voter can set has gauge');
```

Comment:
why are we setting this here? Pairs can be created in the frontend who will we get the value for has gauge passed to the constructor?

I thought hasGauge was set during create gauge (otherwise default to tank?

Reply:
what does "this" mean?

yes, hasGauge is set with createGauage in Voter.

Reply:
yes hasGauge is set with createGauge in Voter